1_fetch/src/fetch.R

```diff
@@ -1,17 +1,10 @@
-library(sbtools)
+#' @param out_filepath string file path under which to save data locally (in out dir)
 
-
-#' @param sb_id character ID corresponding to ScienceBase item
-#' @param src_filename string file name attached to SB item that you wish to download
-#' @param local_filename string file name under which to save data locally (in out dir)
-
-download_sbdata <- function(sb_id, src_filename, local_filename) {
+download_data <- function(out_filepath) {
```

Comment:
Nice work passing in the `out_filepath` as an argument, so that anyone who is looking through `_targets.R` can see what the output file is named and where it is saved. Can you add additional arguments so that we could potentially use this same function to download any specified file from any sb id?

Reply:
Yes, sorry I revised my functions to match the format suggested in the `_targets.R` file, but I guess I should have adapted the `_targets.R` file to match my existing functions instead. I will revert back and update my targets file!

Reply:
Sorry that I missed that! The template was meant to be a starting place, with room for improvement, but I see how that isn't entirely clear. You had it exactly right before.